bigint/src/lib.rs

```diff
@@ -127,6 +127,7 @@ pub mod big_digit {
 
     // `DoubleBigDigit` size dependent
     pub const BITS: usize = 32;
+    pub use ::std::u32::MAX;
```

Comment:
Whoops, fixed.

On Thu, Jul 14, 2016 at 2:12 PM, Josh Stone <notifications@github.com>
wrote:

> In bigint/src/lib.rs
> <https://github.com/rust-num/num/pull/204#discussion_r70893711>:
>
> > @@ -127,6 +127,7 @@ pub mod big_digit {
> >
> >      // `DoubleBigDigit` size dependent
> >      pub const BITS: usize = 32;
> > +    pub use ::std::u32::MAX;
>
> This wasn't addressed in your update.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rust-num/num/pull/204/files/4f92ac79ec2efeb675e24589bd00fdd1efd31906#r70893711>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB5r7kY2E8pQHmYXpoLwKofSfCb1lCLFks5qVrRcgaJpZM4JIeee>
> .
>